docs/navigation.json

```diff
@@ -50,7 +50,15 @@
         "GeoJsonParser",
         "GpxParser",
         "VectorTileParser",
-        "CameraCalibrationParser"
+        "CameraCalibrationParser",
+        "BatchTableParser",
+        "BatchTableHierarchyExtensionParser"
+    ],
+
+    "Provider": [
+        "URLBuilder",
```

Comment:
Don't forget to remove them from above !

Reply:
My bad I didn't see that they were in Source, I'll let them there then.

Reply:
Oh no, I would prefer to move them in `Provider`, as it is the name of the folder they are in.

Reply:
Ok, done